src/Microsoft.VisualBasic/tests/StringsTests.cs

```diff
@@ -174,12 +174,175 @@ public void Filter_Objects_WhenObjectCannotBeConvertedToString_ThrowsArgumentOut
         [Theory]
         [InlineData(new object[] { 42 }, "42", new string[] { "42" }, new string[] { })]
         [InlineData(new object[] { true }, "True", new string[] { "True" }, new string[] { })]
-        public void Objects(object[] source, string match, string[] includeExpected, string[] excludeExpected)
+        public void Filter_Objects(object[] source, string match, string[] includeExpected, string[] excludeExpected)
         {
             Assert.Equal(includeExpected, Strings.Filter(source, match, Include: true));
             Assert.Equal(excludeExpected, Strings.Filter(source, match, Include: false));
         }
 
+        [Theory]
+        [InlineData(null, null, 0)]
+        [InlineData(null, "", 0)]
+        [InlineData("", null, 0)]
+        [InlineData("", "", 0)]
+        [InlineData(null, "a", 0)]
+        [InlineData("a", null, 1)]
+        [InlineData("a", "a", 1)]
+        [InlineData("aa", "a", 1)]
+        [InlineData("ab", "a", 1)]
+        [InlineData("ba", "a", 2)]
+        [InlineData("b", "a", 0)]
+        [InlineData("a", "ab", 0)]
+        [InlineData("ab", "ab", 1)]
+        public void InStr_FromBegin(string string1, string string2, int expected)
+        {
+            Assert.Equal(expected, Strings.InStr(string1, string2));
+            Assert.Equal(expected, Strings.InStr(1, string1, string2));
+        }
+
+        [Theory]
+        [InlineData(null, null, 0)]
+        [InlineData(null, "", 0)]
+        [InlineData("", null, 0)]
+        [InlineData("", "", 0)]
+        [InlineData(null, "a", 0)]
+        [InlineData("aa", null, 2)]
+        [InlineData("aa", "a", 2)]
+        [InlineData("aab", "a", 2)]
+        [InlineData("aba", "a", 3)]
+        [InlineData("ab", "a", 0)]
+        [InlineData("aa", "ab", 0)]
+        [InlineData("abab", "ab", 3)]
+        public void InStr_FromWithin(string string1, string string2, int expected)
+        {
+            var startPos = 2;
```

Comment:
can you please remove var from all of these.
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md
```We only use var when it's obvious what the variable type is (e.g. var stream = new FileStream(...) not var stream = OpenStandardInput()).```